README.md

```diff
@@ -154,12 +153,13 @@ wallet saved in /home/ubuntu/.zcn/wallet.json
 "version":"1.0","date_created":"2023-05-03T12:44:46+05:30","nonce":0}
 
 ```
-Here is a sample with `faucet` command and this creates a wallet at default location`~/.zcn/wallet.json`
+#### Creating wallet with 'faucet' command
+Here is a sample `faucet` command, and this creates a wallet at default location`~/.zcn/wallet.json`
 
 ```sh
 ./zwallet faucet --methodName pour --input "new wallet"
 ```
-
+#### Creating a second wallet with 'faucet' command
```

Comment:
The output for this command shows that this command is finding if a  wallet exists or not then its creating another wallet so its better to allign the command with the heading or change the heading to "Creating a second wallet with 'faucent' command " then its output should not show that the no wallet exists in the path because its task is to create the wallet not to find an existing wallet